### DISCUSSION TITLE
Solidjs

### DISCUSSION THREAD

**Discussion by wobsoriano** [ai/nanostores#48] [General]
Might be a good addition https://github.com/wobsoriano/solid-nanostores

**Comment by ai** [comment]
Do you want move it to Nano Stores org?

**Reply by wobsoriano** [reply]
Yes

**Reply by ai** [reply]
1. I sent you invite. Feel free to move project to `nanostores`.
2. Remove `fork` mark in repo since it hides your repo from the search.
3. Send PR to `nanostores/nanostores` with a link to your project

**Reply by wobsoriano** [reply]
Sorry just now. So I'll transfer ownership to nanostores, then fork the [nanostore repo](https://github.com/nanostores/nanostores) and add link to solid-nanostores under **Solid** section in README right?

**Reply by ai** [reply]
Correct
